src/bindings/python/src/pyopenvino/core/async_infer_queue.cpp

```diff
@@ -359,7 +359,7 @@ void regclass_AsyncInferQueue(py::module m) {
             return self._user_ids;
         },
         R"(
-        :return: List of all passed userdata. None if the data wasn't passed yet.
+        :return: List of all passed userdata. List[None] if the data wasn't passed yet.
```

Comment:
```suggestion
        :return: List of all passed userdata. List is filled with `None` if the data wasn't passed yet.
```
I would suggest something like this, as `List[Any]` and `List[None]` are virtually the same. I think that suggesting specific value (which is in fact there) is better.

Reply:
applied